package.json

```diff
@@ -3,12 +3,40 @@
   "productName": "Sabaki",
   "version": "0.20.0",
   "description": "An elegant Go/Baduk/Weiqi board and SGF editor for a more civilized age.",
-  "author": "Yichuan Shen",
+  "author": "Yichuan Shen <shen.yichuan@gmail.com>",
+  "homepage": "http://sabaki.yichuanshen.de",
+  "license": "MIT",
   "main": "main.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/yishn/Sabaki"
   },
+  "build": {
+    "appId": "de.yichuanshen.sabaki",
+    "copyright": "Copyright © 2015-2017 Yichuan Shen",
+    "linux": {
+      "category": "Game",
+      "packageCategory": "games",
+      "target": ["deb", "rpm", "pacman", "tar.gz"]
+    },
+    "mac": {
+      "category": "public.app-category.board-games"
+    },
+    "win": {
+      "iconUrl": "https://raw.githubusercontent.com/yishn/sabaki/master/data/icon.ico",
+      "target": ["msi"]
+    },
+    "fileAssociations": [
+      {
+        "ext": "sgf",
+        "name": "SGF"
+      }
+    ],
+    "files": [
+      "**/*",
+      "!(build.js|docs|tests?|examples|.c9|scss|^/node_modules/(moment|(octicons|sigma)/(plugins|lib|src|build/font)))($|/)"
```

Comment:
Ideally we can now split this massive rule up into a series of smaller rules, since `build.files` supports that. I just didn't wanna mess with it until I was sure it was correct first.